protos/sources/vega/governance.proto

```diff
@@ -229,6 +231,8 @@ message UpdateMarketConfiguration {
   string tick_size = 12;
   // If enabled aggressive orders sent to the market will be delayed by the configured number of blocks
   bool enable_transaction_reordering = 13;
+  // Number of allowed price levels between an AMM's fair price and its quote prices. An AMM definition that exceeds this will we rejected at submission.
```

Comment:
```suggestion
  // Number of allowed price levels between an AMM's fair price and its quote prices. An AMM definition that exceeds this will be rejected at submission.
```